components/WithIcons.tsx

```diff
@@ -2,7 +2,7 @@ import React, { useContext } from 'react'
 import Null from './Null';
 
 export type IconProps = {
-  className: string;
+  className?: string;
```

Comment:
The `PGDocsLink` does not need a custom `className` so it seems reasonable to make this optional.